src/libstd/sys/unix/fd.rs

```diff
@@ -18,6 +18,11 @@ use sys::cvt;
 use sys_common::AsInner;
 use sys_common::io::read_to_end_uninitialized;
 
+#[cfg(any(target_os = "linux", target_os = "emscripten", target_os = "android"))]
+use libc::{pread64, pwrite64, off64_t};
+#[cfg(not(any(target_os = "linux", target_os = "emscripten", target_os = "android")))]
+use libc::{pread as pread64, pwrite as pwrite64, off_t as off64_t};
```

Comment:
The same thing is done in [src/libstd/sys/unix/fs.rs](https://github.com/rust-lang/rust/blob/289f3a4ca79916d6445b452fc19a18a1e42a879a/src/libstd/sys/unix/fs.rs#L31-L35).